esp-wifi-sys/src/lib.rs

```diff
@@ -12,3 +14,77 @@ pub mod c_types;
 #[cfg_attr(feature = "esp32s2", path = "include/esp32s2.rs")]
 #[cfg_attr(feature = "esp32s3", path = "include/esp32s3.rs")]
 pub mod include;
+
+#[cfg(feature = "sys-logs")]
+pub mod log {
+    #[no_mangle]
+    pub unsafe extern "C" fn rtc_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn phy_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn coexist_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn net80211_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn pp_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn syslog(
+        _priority: u32,
+        _format: *const u8,
```

Comment:
nitpick but since format and args are now always used we can remove the `_` prefix